bin/pubber

```diff
@@ -31,29 +31,26 @@ else
         if [[ $option == *"="* ]]; then
             k=$(echo $option | cut -d'=' -f1)
             v=$(echo $option | cut -d'=' -f2)
-            options[$k]=$v
+            if ! [[ $v =~ ^[0-9]+([.][0-9]+)?$ ]]; then
```

Comment:
I wouldn't go about it this way, since although it seems "simple" it's actually somewhat of a minefield because, say, what if you want a string that looks like a number?  Better to be just simpler (all options are strings) and then deal with the conversion later, if/when necessary.

Reply:
Removed so options are either boolean true or string value